examples/deployment/algob.config.js

```diff
@@ -52,6 +53,28 @@ let accounts = mkAccounts([
   }
 ]);
 
+// KMD credentials
+let KMDConfig = {
+  host: "127.0.0.1",
+  port: 7833,
+  token: "09c2da31d3e3e96ed98ba22cc4d58a14184f1808f2b4f21e66c9d38f70ca7232"
+}
+
+let wallet = {
+  name: "MyTestWallet1",
+  password: "testpassword"
+}
+
+// load KMD Accounts from given wallet - (addr, secret key)
+async function kmdAdd() {
+  const kmdAddresses = await loadKMDAddresses(KMDConfig.host, KMDConfig.token, KMDConfig.port, 
+    wallet.name, wallet.password);
+
+    console.log(kmdAddresses);
+}
+
+kmdAdd();
```

Comment:
here is the problem. You don't wait for the `kmdAdd` competition. This function is asynchronous.

What you can do is:

1. Extend the network type definition and add `kmdAddresses: Promise<AccountSDK[]>` (provisionarily)
1. assign kmd addresses to `default` network:

        defaltCfg.kmdAddresses = loadKMDAddresses(KMDConfig.host, KMDConfig.token, KMDConfig.port, 
    wallet.name, wallet.password);

1. Print kmdAddressess in the `node-info` comand (builtin-tasks).  -- to print, you will need to await first:

        let kmdaddrs = await network.kmdAddresses;

Reply:
I mean, you would need to call `await kmdAdd()` to have this working.